orchestrator/graphql/resolvers/subscription.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019-2020 SURF.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Union
+
+import structlog
+from graphql import GraphQLError
+from sqlalchemy.orm import joinedload
+
+from orchestrator.db import SubscriptionTable
+from orchestrator.db.filters import CallableErrorHander, Filter
+from orchestrator.db.filters.subscription import filter_subscriptions
+from orchestrator.db.range import apply_range_to_query
+from orchestrator.db.sorting import Sort, sort_subscriptions
+from orchestrator.graphql.pagination import Connection, PageInfo
+from orchestrator.graphql.schemas.subscription import SubscriptionType
+from orchestrator.graphql.types import CustomInfo, GraphqlFilter, GraphqlSort
+
+logger = structlog.get_logger(__name__)
+
+
+def handle_subscription_error(info: CustomInfo) -> CallableErrorHander:
+    def _handle_subscription_error(message: str, **kwargs) -> None:  # type: ignore
+        logger.debug(message, **kwargs)
+        extra_values = dict(kwargs.items()) if kwargs else {}
+        info.context.errors.append(GraphQLError(message=message, path=info.path, extensions=extra_values))
+
+    return _handle_subscription_error
+
+
+async def resolve_subscription(
```

Comment:
```suggestion
async def resolve_subscriptions(
```

For consistency with the other resolvers :)